packages/opencensus-instrumentation-http/src/http.ts

```diff
@@ -173,8 +220,8 @@ export class HttpPlugin extends classes.BasePlugin {
 
         plugin.logger.debug('%s plugin outgoingRequest', plugin.moduleName);
         const traceOptions = {
-          name: `${request.method ? request.method : 'GET'}  ${
-              arguments[0].pathname}`,
+          name:
+              `${request.method ? request.method : 'GET'}  ${options.pathname}`,
```

Comment:
Nit: Is the extra space between the two expressions needed?

Reply:
Done by ``gts fix``.
 ``gts check`` fails otherwise.

Reply:
Hmm... I'm wondering about the two spaces in the string, which I don't think has to do with code formatting:
```js
`${request.method ? request.method : 'GET'}  ${options.pathname}`
                                           ^^
```

Reply:
Should be only one space. Done !